As a system admin, I want dependency (library, package, image, configuration) updates for security and bug fixes for WRES 6.26

- [ ] Libraries having update available
- [x] Libraries marked vulnerable by scan
- [x] #88
- [ ] Jetty
- [x] Artemis
- [x] Redis
- [x] RabbitMQ
- [x] Java Runtime Environment

The deployment ticket is #312.  

The above checklist was copied from #67 for the 6.25 deployment. If its different for this deployment, let me know. 

In #67, Arvin noted these dependency updates to be skipped:

https://github.com/NOAA-OWP/wres/issues/68
https://github.com/NOAA-OWP/wres/issues/69 (THIS WAS FIXED!)
https://github.com/NOAA-OWP/wres/issues/70

Unless someone notes otherwise, I'll skip those again.  

Thanks,

Hank

Leave it to me to break the `build.gradle` somehow. When I try `dependencyCheckAnalyze`, it fails. See the message below. Investigating,

Hank

==============

```
> Task :dependencyCheckAnalyze FAILED
Verifying dependencies for project wres

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dependencyCheckAnalyze'.
> Class org.gradle.api.internal.artifacts.configurations.DefaultConfiguration_Decorated does not implement the requested interface org.gradle.api.Named

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

--- a/build.gradle
+++ b/build.gradle
@@ -902,7 +902,7 @@ project(':wres-vis') {
         implementation group: 'org.jfree', name: 'jfreechart', version: '1.5.5'
 
         // SVG graphics from JFreeChart instances
-        implementation group: 'org.jfree', name: 'org.jfree.svg', version: '4.2'
+        implementation group: 'org.jfree', name: 'org.jfree.svg', version: '4.1'
 
         runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat

--- a/build.gradle
+++ b/build.gradle
@@ -315,7 +315,7 @@ project(':wres-datamodel') {
         implementation 'systems.uom:systems-quantity:2.1'
 
         // For geometries
-        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
+        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.20.0'
 
         // JCIP annotations
         compileOnly group: 'net.jcip', name: 'jcip-annotations', version: '1.0'
@@ -386,8 +386,8 @@ project(':wres-io') {
         }
 
         // For geometries
-        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
-        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.19.0', ext: 'pom'
+        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.20.0'
+        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.20.0', ext: 'pom'
 
         implementation 'org.liquibase:liquibase-core:4.29.2'
 
@@ -603,8 +603,8 @@ project(':wres-reading') {
 
 
         // For geometries
-        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
-        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.19.0', ext: 'pom'
+        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.20.0'
+        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.20.0', ext: 'pom'
 
         // Fast-Infoset encoded XML
         implementation group: 'com.sun.xml.fastinfoset', name: 'FastInfoset', version: '2.1.1'
@@ -980,8 +980,8 @@ project(':wres-config') {
         implementation 'com.google.guava:guava:33.3.0-jre'
 
         // To validate WKT strings as geometries
-        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
-        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.19.0', ext: 'pom'
+        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.20.0'
+        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.20.0', ext: 'pom'
 
         // Protobuf utilities
         implementation 'com.google.protobuf:protobuf-java-util:3.21.12'
@@ -1375,7 +1375,7 @@ project(':wres-tasker') {
         implementation 'org.apache.tika:tika-core:2.9.2'
 
         // To persist job metadata while keeping it relatively fast
-        implementation('org.redisson:redisson:3.36.0') {
+        implementation('org.redisson:redisson:3.18.0') {
             // Because of CVE-2022-24823. TODO: remove when Redisson catches up
             exclude group: 'io.netty'
         }
@@ -1894,7 +1894,7 @@ dependencies {
     }
 
     // To persist job metadata while keeping it relatively fast
-    implementation('org.redisson:redisson:3.36.0') {
+    implementation('org.redisson:redisson:3.18.0') {
         // Because of CVE-2022-24823. TODO: remove when Redisson catches up
         exclude group: 'io.netty'
     }

--- a/build.gradle
+++ b/build.gradle
@@ -1347,23 +1347,23 @@ project(':wres-tasker') {
         implementation 'commons-codec:commons-codec:1.17.1'
 
         // Server library to run a web server:
-        implementation 'org.eclipse.jetty:jetty-server:11.0.20'
+        implementation 'org.eclipse.jetty:jetty-server:11.0.24'
 
         // Support HTTP/2
         implementation 'org.eclipse.jetty.http2:http2-server:11.0.24'
 
         // Support ALPN
-        implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.21'
+        implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.24'
 
         // Servlet container library to run a web application with:
         implementation 'org.eclipse.jetty:jetty-webapp:11.0.24'
 
         // Needed at compile-time to reference ServletContainer.class and
         // DefaultServlet.class
-        implementation 'org.glassfish.jersey.containers:jersey-container-servlet-core:3.1.6'
+        implementation 'org.glassfish.jersey.containers:jersey-container-servlet-core:3.1.8'
 
         // To handle multipart post
-        implementation 'org.glassfish.jersey.media:jersey-media-multipart:3.1.6'
+        implementation 'org.glassfish.jersey.media:jersey-media-multipart:3.1.8'
 
         // To temporarily cache/store-in-RAM job metadata when redis unavailable
         implementation('com.github.ben-manes.caffeine:caffeine:3.1.8') {
@@ -1408,11 +1408,11 @@ project(':wres-tasker') {
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
 
-        runtimeOnly 'org.glassfish.jersey.core:jersey-server:3.1.6'
+        runtimeOnly 'org.glassfish.jersey.core:jersey-server:3.1.8'
 
         // Needed to prevent "java.lang.IllegalStateException: InjectionManagerFactory not found."
         // Thanks, https://luohuahuang.org/2017/11/13/jersey-illegalstateexception-injectionmanagerfactory/
-        runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.6'
+        runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.8'
 
         testImplementation 'junit:junit:4.13.2'
         testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.11.0'
@@ -1883,10 +1883,10 @@ dependencies {
 
     // Needed at compile-time to reference ServletContainer.class and
     // DefaultServlet.class
-    implementation 'org.glassfish.jersey.containers:jersey-container-servlet-core:3.1.6'
+    implementation 'org.glassfish.jersey.containers:jersey-container-servlet-core:3.1.8'
 
     // Jetty server
-    implementation 'org.eclipse.jetty:jetty-server:11.0.20'
+    implementation 'org.eclipse.jetty:jetty-server:11.0.24'
 
     implementation('com.github.ben-manes.caffeine:caffeine:3.1.8') {
         // Not used at runtime, bloat
@@ -1918,18 +1918,18 @@ dependencies {
     // JCIP annotations
     compileOnly group: 'net.jcip', name: 'jcip-annotations', version: '1.0'
 
-    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-servlet:3.1.6'
-    runtimeOnly 'org.glassfish.jersey.core:jersey-server:3.1.6'
+    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-servlet:3.1.8'
+    runtimeOnly 'org.glassfish.jersey.core:jersey-server:3.1.8'
     // currently locked at this version, check this ticket for more information
     // https://vlab.***REMOVED***/redmine/issues/124200
-    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-jetty-http:3.1.3'
-    runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.6'
+    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-jetty-http:3.1.8'
+    runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.8'
 
     // Support HTTP/2
     implementation 'org.eclipse.jetty.http2:http2-server:11.0.24'
 
     // Support ALPN
-    implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.20'
+    implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.24'
 
     runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
         // Not used at runtime, bloat

--- a/build.gradle
+++ b/build.gradle
@@ -62,7 +62,7 @@ plugins {
     id 'com.adarshr.test-logger' version '4.0.0'
 
     // To discover library versions with known vulnerabilities
-    id 'org.owasp.dependencycheck' version '10.0.4'
+    id 'org.owasp.dependencycheck' version '10.0.2'
 
     // Task to assist in downloading artifacts
     id 'de.undercouch.download' version '5.6.0'

--- a/build.gradle
+++ b/build.gradle
@@ -1922,7 +1922,7 @@ dependencies {
     runtimeOnly 'org.glassfish.jersey.core:jersey-server:3.1.8'
     // currently locked at this version, check this ticket for more information
     // https://vlab.***REMOVED***/redmine/issues/124200
-    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-jetty-http:3.1.8'
+    runtimeOnly 'org.glassfish.jersey.containers:jersey-container-jetty-http:3.1.3'
     runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.8'
 
     // Support HTTP/2

--- a/build.gradle
+++ b/build.gradle
@@ -79,7 +79,7 @@ plugins {
     id 'org.kordamp.gradle.markdown' version '2.2.0'
 
     // To generate API documentation from JAX-RS annotations and javadocs
-    id "io.swagger.core.v3.swagger-gradle-plugin" version "2.2.22"
+    id "io.swagger.core.v3.swagger-gradle-plugin" version "2.2.23"
 
     // To generate JPMS images more easily with 3rd-party "automatic" modules
     id 'org.beryx.jlink' version '2.26.0'
@@ -238,7 +238,7 @@ project(':wres-system') {
     dependencies {
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        implementation 'org.apache.commons:commons-lang3:3.16.0'
+        implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // to use native postgres copy, need this on compile, otherwise runtime
         implementation('org.postgresql:postgresql:42.7.3') {
@@ -292,11 +292,11 @@ project(':wres-datamodel') {
         implementation project(':wres-config')
         api project(':wres-statistics')
 
-        api 'org.apache.commons:commons-lang3:3.16.0'
+        api 'org.apache.commons:commons-lang3:3.17.0'
 
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -666,7 +666,7 @@ project(':wres-metrics') {
 
         compileOnly 'net.jcip:jcip-annotations:1.0'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -790,7 +790,7 @@ project(':wres-statistics') {
 
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -894,7 +894,7 @@ project(':wres-vis') {
         implementation project(':wres-eventsbroker')
         implementation project(':wres-events')
 
-        implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.14.0'
+        implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.17.0'
         implementation group: 'com.sun.xml.fastinfoset', name: 'FastInfoset', version: '2.1.1'
 
         implementation 'org.slf4j:slf4j-api:2.0.13'
@@ -904,7 +904,7 @@ project(':wres-vis') {
         // SVG graphics from JFreeChart instances
         implementation group: 'org.jfree', name: 'org.jfree.svg', version: '4.1'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -943,7 +943,7 @@ project(':wres-config') {
         implementation project(':wres-statistics')
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -986,7 +986,7 @@ project(':wres-config') {
         // Protobuf utilities
         implementation 'com.google.protobuf:protobuf-java-util:3.21.12'
 
-        implementation 'org.apache.commons:commons-lang3:3.16.0'
+        implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // For reading CSV thresholds
         implementation group: 'com.opencsv', name: 'opencsv', version: '5.9'
@@ -1192,7 +1192,7 @@ project(':wres-worker') {
         implementation 'org.slf4j:slf4j-api:2.0.13'
         implementation 'com.rabbitmq:amqp-client:5.21.0'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -1398,12 +1398,12 @@ project(':wres-tasker') {
         implementation 'com.fasterxml.jackson.core:jackson-core:2.17.2'
 
         // For ToStringBuilder
-        implementation 'org.apache.commons:commons-lang3:3.16.0'
+        implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // To send/receive messages
         implementation 'com.google.protobuf:protobuf-java:3.21.12'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -1431,7 +1431,7 @@ project(':wres-tasker') {
         testImplementation group: 'com.google.jimfs', name: 'jimfs', version: '1.3.0'
 
         // To generate swagger's openapi.json at build time.
-        docGen 'io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.22'
+        docGen 'io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.23'
     }
 
     apply plugin: 'application'
@@ -1503,13 +1503,13 @@ project(':wres-eventsbroker') {
     dependencies {
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        implementation 'org.apache.commons:commons-lang3:3.16.0'
+        implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // Jakarta Messaging API
         implementation group: 'jakarta.jms', name: 'jakarta.jms-api', version: '3.1.0'
 
         // Apache ActiveMQ Artemis server
-        implementation('org.apache.activemq:artemis-server:2.36.0') {
+        implementation('org.apache.activemq:artemis-server:2.37.0') {
             // TODO: remove when Artemis catches up
 
             // Because we use slf4j, not jcl:
@@ -1538,7 +1538,7 @@ project(':wres-eventsbroker') {
         implementation group: 'org.bouncycastle', name: 'bcprov-jdk18on', version: '1.78.1'
 
         // AMQP 1.0 protocol support for the ActiveMQ Artemis server
-        implementation('org.apache.activemq:artemis-amqp-protocol:2.36.0') {
+        implementation('org.apache.activemq:artemis-amqp-protocol:2.37.0') {
             // Because we use slf4j, not jcl:
             exclude group: 'commons-logging', module: 'commons-logging'
 
@@ -1549,7 +1549,7 @@ project(':wres-eventsbroker') {
         // JCIP annotations
         compileOnly group: 'net.jcip', name: 'jcip-annotations', version: '1.0'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -1602,9 +1602,9 @@ project(':wres-events') {
         implementation group: 'io.netty', name: 'netty-all', version: '4.1.111.Final'
 
         // For various utilities
-        implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.14.0'
+        implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.17.0'
 
-        runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+        runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
@@ -1635,7 +1635,7 @@ project(':wres-http') {
         // Better-than-Java's HTTP client
         implementation 'com.squareup.okhttp3:okhttp:4.12.0'
 
-        implementation 'org.apache.commons:commons-lang3:3.16.0'
+        implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // Update vulnerable kotlin-stdlib that okhttp/okio uses
         implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.9.23'
@@ -1931,7 +1931,7 @@ dependencies {
     // Support ALPN
     implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.20'
 
-    runtimeOnly('ch.qos.logback:logback-classic:1.5.7') {
+    runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
         // Not used at runtime, bloat
         exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
     }

--- a/build.gradle
+++ b/build.gradle
@@ -62,7 +62,7 @@ plugins {
     id 'com.adarshr.test-logger' version '4.0.0'
 
     // To discover library versions with known vulnerabilities
-    id 'org.owasp.dependencycheck' version '10.0.2'
+    id 'org.owasp.dependencycheck' version '10.0.4'
 
     // Task to assist in downloading artifacts
     id 'de.undercouch.download' version '5.6.0'
@@ -241,7 +241,7 @@ project(':wres-system') {
         implementation 'org.apache.commons:commons-lang3:3.17.0'
 
         // to use native postgres copy, need this on compile, otherwise runtime
-        implementation('org.postgresql:postgresql:42.7.3') {
+        implementation('org.postgresql:postgresql:42.7.4') {
             // Not used at runtime, bloat
             exclude group: 'org.checkerframework', module: 'checker-qual'
         }
@@ -321,7 +321,7 @@ project(':wres-datamodel') {
         compileOnly group: 'net.jcip', name: 'jcip-annotations', version: '1.0'
 
         // Mocking help
-        testImplementation 'org.mockito:mockito-core:5.12.0'
+        testImplementation 'org.mockito:mockito-core:5.13.0'
 
         // JUnit 5 API and runtime
         testImplementation 'org.junit.jupiter:junit-jupiter-api:5.11.0'
@@ -362,7 +362,7 @@ project(':wres-io') {
         implementation 'com.google.guava:guava:33.3.0-jre'
 
         // to use native postgres copy, need this on compile, otherwise runtime
-        implementation('org.postgresql:postgresql:42.7.3') {
+        implementation('org.postgresql:postgresql:42.7.4') {
             // Not used at runtime, bloat
             exclude group: 'org.checkerframework', module: 'checker-qual'
         }
@@ -389,7 +389,7 @@ project(':wres-io') {
         implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
         implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.19.0', ext: 'pom'
 
-        implementation 'org.liquibase:liquibase-core:4.29.1'
+        implementation 'org.liquibase:liquibase-core:4.29.2'
 
         // Use instead of the bridge between JUL and SLF4J. #60801-283
         runtimeOnly 'com.mattbertolini:liquibase-slf4j:5.0.0'
@@ -546,8 +546,8 @@ project(':wres-writing') {
         implementation 'org.apache.tika:tika-core:2.9.2'
 
         // For geometries
-        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.19.0'
-        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.19.0', ext: 'pom'
+        implementation group: 'org.locationtech.jts', name: 'jts-core', version: '1.20.0'
+        implementation group: 'org.locationtech.jts', name: 'jts-io', version: '1.20.0', ext: 'pom'
 
         compileOnly 'net.jcip:jcip-annotations:1.0'
 
@@ -611,7 +611,7 @@ project(':wres-reading') {
 
         implementation group: 'org.apache.commons', name: 'commons-collections4', version: '4.4'
 
-        implementation 'org.liquibase:liquibase-core:4.29.1'
+        implementation 'org.liquibase:liquibase-core:4.29.2'
 
         // Use instead of the bridge between JUL and SLF4J. #60801-283
         runtimeOnly 'com.mattbertolini:liquibase-slf4j:5.0.0'
@@ -671,7 +671,7 @@ project(':wres-metrics') {
             exclude group: 'edu.washington.cs.types.checker', module: 'checker-framework'
         }
 
-        testImplementation 'org.mockito:mockito-core:5.12.0'
+        testImplementation 'org.mockito:mockito-core:5.13.0'
 
         // JUnit 5 API and runtime
         testImplementation 'org.junit.jupiter:junit-jupiter-api:5.11.0'
@@ -899,10 +899,10 @@ project(':wres-vis') {
 
         implementation 'org.slf4j:slf4j-api:2.0.13'
 
-        implementation group: 'org.jfree', name: 'jfreechart', version: '1.5.4'
+        implementation group: 'org.jfree', name: 'jfreechart', version: '1.5.5'
 
         // SVG graphics from JFreeChart instances
-        implementation group: 'org.jfree', name: 'org.jfree.svg', version: '4.1'
+        implementation group: 'org.jfree', name: 'org.jfree.svg', version: '4.2'
 
         runtimeOnly('ch.qos.logback:logback-classic:1.5.8') {
             // Not used at runtime, bloat
@@ -921,7 +921,7 @@ project(':wres-vis') {
         testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.11.0'
 
         // Mocking help
-        testImplementation 'org.mockito:mockito-core:5.12.0'
+        testImplementation 'org.mockito:mockito-core:5.13.0'
 
         testRuntimeOnly files('dist/lib/conf')
 
@@ -1025,7 +1025,7 @@ project(':wres-config') {
         implementation group: 'org.jvnet.jaxb2_commons', name: 'jaxb2-basics-annotate', version: '1.1.0'
 
         // Mocking help
-        testImplementation 'org.mockito:mockito-core:5.12.0'
+        testImplementation 'org.mockito:mockito-core:5.13.0'
 
         testImplementation 'org.junit.jupiter:junit-jupiter-api:5.11.0'
         testImplementation 'org.junit.jupiter:junit-jupiter-params:5.11.0'
@@ -1350,13 +1350,13 @@ project(':wres-tasker') {
         implementation 'org.eclipse.jetty:jetty-server:11.0.20'
 
         // Support HTTP/2
-        implementation 'org.eclipse.jetty.http2:http2-server:11.0.22'
+        implementation 'org.eclipse.jetty.http2:http2-server:11.0.24'
 
         // Support ALPN
         implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.21'
 
         // Servlet container library to run a web application with:
-        implementation 'org.eclipse.jetty:jetty-webapp:11.0.22'
+        implementation 'org.eclipse.jetty:jetty-webapp:11.0.24'
 
         // Needed at compile-time to reference ServletContainer.class and
         // DefaultServlet.class
@@ -1375,7 +1375,7 @@ project(':wres-tasker') {
         implementation 'org.apache.tika:tika-core:2.9.2'
 
         // To persist job metadata while keeping it relatively fast
-        implementation('org.redisson:redisson:3.18.0') {
+        implementation('org.redisson:redisson:3.36.0') {
             // Because of CVE-2022-24823. TODO: remove when Redisson catches up
             exclude group: 'io.netty'
         }
@@ -1614,7 +1614,7 @@ project(':wres-events') {
         testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.11.0'
 
         // Mocking help
-        testImplementation 'org.mockito:mockito-core:5.12.0'
+        testImplementation 'org.mockito:mockito-core:5.13.0'
         // Mocking final classes
         testImplementation 'org.mockito:mockito-inline:5.2.0'
 
@@ -1879,7 +1879,7 @@ dependencies {
     implementation 'jakarta.xml.bind:jakarta.xml.bind-api:3.0.1'
 
     // Servlet container library to run a web application with:
-    implementation 'org.eclipse.jetty:jetty-webapp:11.0.22'
+    implementation 'org.eclipse.jetty:jetty-webapp:11.0.24'
 
     // Needed at compile-time to reference ServletContainer.class and
     // DefaultServlet.class
@@ -1894,7 +1894,7 @@ dependencies {
     }
 
     // To persist job metadata while keeping it relatively fast
-    implementation('org.redisson:redisson:3.18.0') {
+    implementation('org.redisson:redisson:3.36.0') {
         // Because of CVE-2022-24823. TODO: remove when Redisson catches up
         exclude group: 'io.netty'
     }
@@ -1926,7 +1926,7 @@ dependencies {
     runtimeOnly 'org.glassfish.jersey.inject:jersey-hk2:3.1.6'
 
     // Support HTTP/2
-    implementation 'org.eclipse.jetty.http2:http2-server:11.0.22'
+    implementation 'org.eclipse.jetty.http2:http2-server:11.0.24'
 
     // Support ALPN
     implementation 'org.eclipse.jetty:jetty-alpn-java-server:11.0.20'
@@ -1950,7 +1950,7 @@ dependencies {
     testCompileOnly 'junit:junit:4.13.2'
     testRuntimeOnly 'org.junit.vintage:junit-vintage-engine:5.11.0'
 
-    testImplementation 'org.mockito:mockito-core:5.12.0'
+    testImplementation 'org.mockito:mockito-core:5.13.0'
 
     testImplementation group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
 }

--- a/wres-broker/Dockerfile
+++ b/wres-broker/Dockerfile
@@ -1,4 +1,4 @@
-FROM rabbitmq:3.13.6-management-alpine
+FROM rabbitmq:3.13.7-management-alpine
 
 # Wish we could do the following, but busybox does not support long gids:
 #RUN addgroup -g 1370800073 wres && adduser -D -u 498 -g 1370800073 wres_docker

--- a/wres-eventsbroker/Dockerfile
+++ b/wres-eventsbroker/Dockerfile
@@ -2,9 +2,9 @@
 # Build image
 #############
 
-FROM alpine:3.19.1 AS builder
+FROM alpine:3.20.3 AS builder
 
-ARG BROKER_VERSION=2.36.0
+ARG BROKER_VERSION=2.37.0
 
 WORKDIR /workspace
 

